arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/raise/Raise.kt

```diff
@@ -249,6 +250,10 @@ public interface Raise<in Error> {
   @RaiseDSL
   public fun <A> Iterable<Either<Error, A>>.bindAll(): List<A> =
     map { it.bind() }
+
+  @RaiseDSL
+  public fun <A> NonEmptyList<Either<Error, A>>.bindAll(): NonEmptyList<A> =
```

Comment:
If we are including `NonEmptyList`, should we also include `Set` and other types of collections?

Reply:
Yes, I thought about adding `NonEmptySet` too today